### ISSUE TITLE
Whitelist Rules does not work

### ISSUE THREAD

**Issue by ddd28et** [bitbeans/SimpleDnsCrypt#564]
I need to block all websites except those are allowed in my whitelist. So I create rules:

Whitelist Rules
`medium.com`

BlackList Rules
`*.*`

I can not open medium.com. Is it possible to  block all websites except those are allowed in my whitelist with SimpleDnsCrypt?

**Comment by ddd28et** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/105601114/178094702-34edf4eb-8771-4c19-a412-a80f3921d42a.png)

It's my screenshot. I can not open medium.com

**Comment by ddd28et** [comment]
I have found a solution. Need to delete Simple DNSCrypt and install dnscrypt-proxy. I can do this thing with dnscrypt and it's possible to block all websites except those are allowed in my whitelist with dnscrypt-proxy.

**Comment by GeniusBroccoli** [comment]
> Need to delete Simple DNSCrypt and install dnscrypt-proxy. I can do this thing with dnscrypt

wait wait wait, can you explain, delete dnscrypt and install dnscrypt-proxy with dnscrypt, but... you delete dnscrypt

**Comment by ddd28et** [comment]
1. I deleted Simple DNSCrypt.
2. I install dnscrypt-proxy here: https://github.com/DNSCrypt/dnscrypt-proxy
3. 3. I use dnscrypt-proxy without Simple DNSCrypt.
4. All works.

**Comment by ddd28et** [comment]
> > Need to delete Simple DNSCrypt and install dnscrypt-proxy. I can do this thing with dnscrypt
> 
> wait wait wait, can you explain, delete dnscrypt and install dnscrypt-proxy with dnscrypt, but... you delete dnscrypt

 So it's possible to block all websites except those are allowed in my whitelist with dnscrypt-proxy (new version). Maybe it's  possible with Simple DNSCrypt. I don't know. But I use dnscrypt-proxy here https://github.com/DNSCrypt/dnscrypt-proxy without GUI and it's work fine. I like it.

**Comment by GeniusBroccoli** [comment]
> But I think that dnscrypt-proxy here https://github.com/DNSCrypt/dnscrypt-proxy works very well without any GUI. This program has a very good documentation.

Yes, but Simple DNSCrypt is simple and user-friendly - ready to go solution without studying the documentation, and the concept of the new application looked great, sad if the development is abandoned.
Is there an auto-selection of the best DNS in dnscrypt-proxy by the way?
